app/views/static/performance.html.haml

```diff
@@ -0,0 +1,23 @@
+- content_for :page_title, "Performance"
+
+%h1= yield :page_title
+
+%h2 Comments on planning applications
+
+- if Comment.any?
+  %table
+    %caption
+      New and returning people commenting on applications over time
+    %thead
+      %tr
+        %th Date
+        %th First time commenters
+        %th Returning commenters
+    %tbody
+      - (3.months.ago.to_date..Date.today).to_a.reverse.each do |day|
```

Comment:
Yep I agree. It might be quite clear to have a performance controller for this page.